test-data/unit/check-based-default-return.test

```diff
@@ -101,7 +101,7 @@ class A:
 
 f = lambda x: x
 g = lambda: ...
-reveal_type(f)  # N: Revealed type is "def (x: Any) -> Any"
+reveal_type(f)  # N: Revealed type is "def (x: Untyped) -> Any (special form)"
```

Comment:
id suggest not showing this internal type thing to the user, maybe hide it behing a verbose mode?

Reply:
-> `(unknwon)`

Reply:
It copped up in a lot of sus locations (when it should have been 'Any Explicit' for egg), so I removed it. and it shows the same as explicit now.